apps/extension/src/background/approvals/handler.test.ts

```diff
@@ -48,26 +44,18 @@ describe("approvals handler", () => {
       requestInteraction: () => {},
     };
 
-    const approveTxMsg = new ApproveTxMsg(
-      TxType.Bond,
-      [
-        {
-          txMsg: "txMsg",
-          specificMsg: "specificMsg",
-        },
-      ],
-      AccountType.Mnemonic
-    );
-    handler(env, approveTxMsg);
-    expect(service.approveTx).toBeCalled();
+    // const approveTxMsg = new ApproveTxMsg(
```

Comment:
Can we remove this comment block?

Reply:
I was planning to re-enable this test as `ApproveTxMsg` has changed